Formula/msc-generator.rb

```diff
@@ -0,0 +1,56 @@
+class MscGenerator < Formula
+  desc "Draws signalling charts from textual description"
+  homepage "https://sourceforge.net/p/msc-generator"
+  url "https://downloads.sourceforge.net/project/msc-generator/msc-generator/v6.x/msc-generator-6.3.9.tar.gz"
+  sha256 "87bcb87109dc7ddabbddd68ac361a12ad4f40d7787542d82411e85b61e53d814"
+  license "AGPL-3.0-or-later"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "bison" => :build
+  depends_on "help2man" => :build
+  depends_on "pkg-config" => :build
+  depends_on "cairo"
+  depends_on "graphviz"
+
+  # A single change to move out inline definition of a templated member function -- GCC can do it
+  patch :p1, :DATA
+
+  def install
+    ENV["PATH"] = "#{ENV["HOMEBREW_OPT"]}/gcc/bin:#{ENV["PATH"]}"
+    system "./configure",
+             "CPPFLAGS=-I#{ENV["HOMEBREW_OPT"]}/glpk/include",
+             "--prefix=#{prefix}"
+    # Dance around upstream trying to build everything in doc/ which we don't do for now
+    # system "make", "install"
+    system "make", "-C", "src", "install"
+    system "make", "-C", "doc", "msc-gen.1"
+    man1.install "doc/msc-gen.1"
+  end
+
+  test do
+    system "#{bin}/msc-gen", "--version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Will add a real one.